common/app/filters/RequestLoggingFilter.scala

```diff
@@ -23,7 +23,7 @@ class RequestLoggingFilter extends Filter with Logging with ExecutionContexts {
         "req.method" -> rh.method,
         "req.url" -> rh.uri,
         "req.id" -> pseudoId.toString,
-        "req.latency_ms" -> stopWatch.elapsed.toString
+        "req.latency_millis" -> stopWatch.elapsed
```

Comment:
Renaming the field because change a field type in ES requires a re-indexing which is quite a costly operation. The field is not used anywhere yet so it doesn't cost anything to add a new field.